Assets/Scripts/Audio/PerformanceAudioController.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine;
+
+namespace Audio
+{
+    public class PerformanceAudioController : MonoBehaviour
+    {
+        private AudioBuilderSystem _audioBuilder;
+        private Chapter _chapter;
+
+        [SerializeField] private AudioClip testClip1;
+
+        private void OnEnable()
+        {
+            StagePosition.OnStagePositionChanged += StagePositionChanged;
+        }
+
+        private void OnDisable()
+        {
+            StagePosition.OnStagePositionChanged -= StagePositionChanged;
+        }
+
+        private void Start()
+        {
+            _chapter = FindObjectOfType<Chapter>();
+            if (!_chapter)
+            {
+                StSDebug.LogError($"PerformanceAudioController could not find chapter object.");
+            }
+            
+            _audioBuilder = FindObjectOfType<AudioBuilderSystem>();
+            if (!_audioBuilder)
+            {
+                StSDebug.LogError($"PerformanceAudioController could not find audioBuilder object.");
+            }
+        }
+
+        private void StagePositionChanged(StagePosition obj)
+        {
```

Comment:
Here is the chaos - we require some functionality to accept a musician proficiency, instrument and chapter in as a parameter that returns a corresponding track. Could even be a static system that handles all tracks in one single place somewhere?

Reply:
All tracks as in multiple chapters, or one for each act, or would this be chapter specific?

Could even have scriptable objects of each instrument in a track, something like:
- Instrument
- Track Name
- Perfect Audio
- Good Audio
- Bad Audio

Reply:
I'm happy to handle it however you think makes sense. It almost makes sense to me that we collate it all in one file but I guess that might become chaotic and unmanageable? 
So maybe we could have it ad a giant SO that contains an array of objects representing each act
And each act (SO) has it's own set of data which is pulled from
Can even go further to chapter by chapter but then we have a lot of duplicate data if an instrument is used that isn't part of that chapter's music track ? 
I think Act by Act is enough -  so Act 1 is index 0 in the global data index

Reply:
Win win for this - it might be very easy to make these into prefabs/assetbundles in the future since this is where a lot of the biggest data files will be referened